### PR TITLE
update `README.md` (fixes #960)

### LINKED ISSUE #960: README.md no dev section

### PR DESCRIPTION
**Pull request by LordJashin32**
- [x] Lists files that need to be updated usually for a new feature
https://github.com/treehouses/cli/tree/dev-help

https://github.com/treehouses/cli/blob/dev-help/README.md#L116-L126

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,3 +113,14 @@ log <0|1|2|3|4|show|max>                  gets/sets log level and shows log
 blocker <0|1|2|3|4||max>                  website blocking levels using /etc/hosts
 sdbench                                   displays read and write speed of micro SD card
 ```
+## Add a feature
+Feel free to make a pull request to our repository
+Remember the following files
```
Comment (kjong) [inline README.md:118]: Either need a period after "repository" or an additional line break between "repository and "Remember".

Comment (LordJashin32) [reply README.md:118]: Nice suggestion.

**Comment by vmnet8** [comment]
It is a good reminder note for everyone. Just one question, is the test step mandatory?

**Comment by LordJashin32** [comment]
> It is a good reminder note for everyone. Just one question, is the test step mandatory?

No. But shhhhhh. we want tests hahaha.

**Comment by vmnet8** [comment]
> > It is a good reminder note for everyone. Just one question, is the test step mandatory?
> 
> No. But shhhhhh. we want tests hahaha.

I think testing is good, but my concern is how to write the test.bats :p

**Pull request merged**